Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -682,19 +696,26 @@ namespace AtomSampleViewer
                     }
                 }
                 ImGui::SameLine();
-                if (ImGui::Button("Next"))
+                if (ImGui::Button("Add and Next"))
                 {
-                    m_wizardSettings.m_reportIndexDifferenceLevelMap[currentIndex] = PrecommitWizardSettings::ImageDifferenceLevel(m_wizardSettings.m_inspectionSelection);
+                    m_wizardSettings.m_reportIndexDifferenceLevelMap[currentIndex] = PrecommitWizardSettings::ImageDifferenceLevel::Levels(m_wizardSettings.m_inspectionSelection);
                     m_wizardSettings.m_exportedPngPath.clear();
                     ++m_wizardSettings.m_reportIterator;
 
                     if (m_wizardSettings.m_reportIterator == m_wizardSettings.m_reportsOrderedByThresholdToInspect.end())
                     {
+                        // Decrement the iterator to make sure it's pointing to the last element in case the user goes
+                        // back to the manual inspection page.
                         --m_wizardSettings.m_reportIterator;
                         m_wizardSettings.m_stage = PrecommitWizardSettings::Stage::REPORT_FINAL_SUMMARY;
                     }
                 }
                 ImGui::SameLine();
+                if (ImGui::Button("Finish Manual Inspection"))
+                {
+                    m_wizardSettings.m_stage = PrecommitWizardSettings::Stage::REPORT_FINAL_SUMMARY;
+                }
```

Comment:
What's going to happen if the user clicks this button when there are still pending manual comparisons to make? Will it be clear in the final report that they skipped something?

Reply:
If the user clicks on "Finish", the final summary will include reports on only the manually inspected ones. I was thinking it would be up to the users to click on "Finish" when the threshold is low enough and that they are no longer seeing any major changes (example - Say there are 100 passed screenshots, and they only go through the first 25 before clicking "Finish")

I can add a summary of the number of screenshots manually validated over the total number of passed screenshots if that is useful.